saml/gatein-saml-plugin/src/main/java/org/gatein/sso/saml/plugin/SimpleGroup.java

```diff
@@ -77,27 +75,8 @@ public boolean addMember(Principal user)
     @return true if the principal is a member of this group,
     false otherwise.
     */
-   public boolean isMember(Principal member)
-   {
-      // First see if there is a key with the member name
-      boolean isMember = members.containsKey(member);
-
-      if( isMember == false )
-      {   // Check any Groups for membership
-         Collection values = members.values();
-         Iterator iter = values.iterator();
-         while( isMember == false && iter.hasNext() )
-         {
-            Object next = iter.next();
-            if( next instanceof Group )
-            {
-               Group group = (Group) next;
-               isMember = group.isMember(member);
-            }
-         }
-      }
-
-      return isMember;
```

Comment:
shouldn't we keep this logic ? a principal could have a list of principal of type simpleGroup that may contain the current user

Reply:
done